src/Mvc/Mvc.Core/src/PublicAPI.Unshipped.txt

```diff
@@ -1632,7 +1632,7 @@ Microsoft.AspNetCore.Mvc.ApplicationModels.ActionModel.Controller.set -> void
 Microsoft.AspNetCore.Mvc.ApplicationModels.ActionModel.DisplayName.get -> string!
 Microsoft.AspNetCore.Mvc.ApplicationModels.ActionModel.Filters.get -> System.Collections.Generic.IList<Microsoft.AspNetCore.Mvc.Filters.IFilterMetadata!>!
 Microsoft.AspNetCore.Mvc.ApplicationModels.ActionModel.Parameters.get -> System.Collections.Generic.IList<Microsoft.AspNetCore.Mvc.ApplicationModels.ParameterModel!>!
-Microsoft.AspNetCore.Mvc.ApplicationModels.ActionModel.Properties.get -> System.Collections.Generic.IDictionary<object!, object!>!
+Microsoft.AspNetCore.Mvc.ApplicationModels.ActionModel.Properties.get -> System.Collections.Generic.IDictionary<object!, object?>!
```

Comment:
Missing `*REMOVE*` for this file?

Reply:
They were updated to support nullability earlier, so the REMOVE exists somewhere in this file.